src/table.scss

```diff
@@ -11,32 +11,29 @@
 $block: #{$fd-namespace}-table;
 
 .#{$block} {
+  @mixin fd-trigger-element() {
+    @include fd-active() {
+      background-color: var(--sapList_Active_Background);
+      border-bottom-color: var(--sapList_SelectionBorderColor);
+      color: var(--sapList_Active_TextColor);
+
+      .#{$block}__icon {
+        color: var(--sapContent_ContrastIconColor);
+      }
+
+      @include fd-hover() {
+        background-color: var(--sapList_Active_Background);
+      }
+    }
+  }
+
   // LOCAL VARS (set all themeable properties, always include !default)
-  $fd-table-border-color: fd-color("neutral", 2) !default;
-  $fd-table-border-width: 1px !default;
-  $fd-table-link-color: fd-color(action, 1) !default;
-  $fd-table-background-color: fd-color("background", 2);
-  $fd-table-header-color: fd-color("text", 3) !default;
-  $fd-table-header-background-color: fd-color("neutral", 1) !default;
-  $fd-table-row-background-color--hover: fd-color-state("hover") !default;
-  $fd-table-row-background-color--selected: fd-color-state("selected") !default;
-
-  $fd-table-cell-spacing: $fd-width--gutter !default;
-
-  --fd-table-cell-spacing: var(--fd-width-gutter);
-
-  $fd-table-cell-padding: fd-space("s") !default;
-  $fd-table-header-cell-padding: fd-space(3) !default;
-  $fd-table-transition-params: $fd-animation-speed ease-in !default;
-  $fd-table-sort-icon-size: fd-space(3);
-  $fd-table-sort-column-header-background-color: fd-color("neutral", 2);
-  $fd-table-fixed-column-cell-color: fd-color("background", 2);
-
-  --fd-table-border-color: var(--fd-color-neutral-2);
-  --fd-table-background-color: var(--fd-color-background-2);
-  --fd-table-link-color: var(--fd-color-action-1);
-  --fd-table-header-color: var(--fd-color-text-3);
-  --fd-table-header-background-color: var(--fd-color-neutral-1);
+
+  $fd-table-cell-padding: 0.5rem;
+  $fd-table-header-height: 2.75rem;
```

Comment:
Weird name for this variable because it is only used in cell selectors

Reply:
Naming changed